clients/buckets/bucket.go

```diff
@@ -369,7 +381,7 @@ func (c Client) upsert(ctx context.Context, bucketName string, data []byte) (Res
 	if b, err := unmarshalJSON(apiErr.Body); err != nil && b.Status == "deleting" {
 		logger.V(1).Info(fmt.Sprintf("Bucket '%s' is being deleted. Waiting before re-creation...", b.BucketName))
 		if err := c.awaitBucketRemoved(ctx, bucketName); err != nil {
-			return Response{}, err
+			return api.Response{}, err
 		}
 		return c.Create(ctx, bucketName, data)
 	}
```

Comment:
This is dead code, right? First, it should be `err == nil`, and second, If there is an error during create (conflict), this will lead to this error
![image](https://github.com/user-attachments/assets/0225f60e-c781-48e1-95ab-62dca4d24149)

Reply:
Removed it